manual/program/reviewer_guidelines.rst

```diff
@@ -0,0 +1,167 @@
+
+- Interrupt your bias—refresh yourself after each section as to what rubric/metrics you've established and make sure that you're reviewing a submission accordingly. Be polite. Instead of telling, ask new ways of approach.
+
+- Don't be rude. Always say at least one good thing.
+
+- Be grateful. Point out what you learned and say thanks.
+
+- Stay fresh while reading. Long time reviews might cause some spots to be left uncovered.
+
+- Reviewers have multiple demands on their time, and are frequently under time-constraints to finish reviews, be aware that cognitive biases are efficient, time pressure promotes their influence on decision-making.[4]
+
+
+Other interesting links on the topic about gender bias:
+
+
+`Threats to objectivity in peer review: the case of gender <https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4552397/&sa=D&ust=1490135409272000&usg=AFQjCNHHxEg8S2lRH-uZYvHLWvQquG3fdg>`_:
+
+`A Linguistic Comparison of Letters of Recommendation for Male and Female Chemistry and Biochemistry Job Applicants <https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2572075/&sa=D&ust=1490135409273000&usg=AFQjCNG7ODzPnsgxZEaDoxNdNSfjeVskAw>`_:
+
+`Science faculty's subtle gender biases favor male students <https://www.google.com/url?q=http://www.pnas.org/content/109/41/16474.abstract&sa=D&ust=1490135409274000&usg=AFQjCNHeGWyMB7QNHg67j-Blecszk7H9Ig>`_:
+
+`How stereotypes impair women's careers in science <https://www.google.com/url?q=http://www.pnas.org/content/111/12/4403.abstract0&sa=D&ust=1490135409275000&usg=AFQjCNGs6y8FQwel92fbeClDr8_iTHsxUw>`_:
+
+Prepared by the SciPy 2017 Diversity Committee and Program Chairs. Thanks to Philip B. Stark for useful links.
+
+This section is a compilation from different sources listed below:
+
+- https://doughellmann.com/blog/2011/10/18/how-i-review-a-pycon-talk-proposal/
+
+- https://nearsoft.com/blog/europython-2016-a-review/
+
+- Everyday Bias: Identifying and Navigating Unconscious Judgments in Our Daily Lives. Howard J. Ross.
+
+- https://hbr.org/2015/04/3-ways-to-make-less-biased-decisions
+
+- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4238945/#!po=88.4615
+
+
+[1] http://bmjopen.bmj.com/content/5/9/e008707
+
+[2] http://bjp.rcpsych.org/content/176/1/47/full-text.pdf+html
+
+[3] https://www.ncbi.nlm.nih.gov/pubmed/28580134/ 
+
+[4] https://www.amazon.com/Everyday-Bias-Identifying-Navigating-Unconscious/dp/1442230835
+
+[5] https://implicit.harvard.edu/implicit/takeatest.html
+
+[6] https://hbr.org/2015/04/3-ways-to-make-less-biased-decisions
+
+[7] Threats to objectivity in peer review: the case of gender
```

Comment:
If this is https://doi.org/10.1016%2Fj.tips.2014.06.005 can we use the DOI hyperlink for this?

```suggestion
[7] `Threats to objectivity in peer review: the case of gender <https://doi.org/10.1016%2Fj.tips.2014.06.005>`_
```